pkgs/misc/emulators/dolphin-emu/default.nix

```diff
@@ -22,6 +22,12 @@ stdenv.mkDerivation rec {
       name = "dolphin-emu-5.0-soundtouch-exception-fix.patch";
       sha256 = "0yd3l46nja5qiknnl30ryad98f3v8911jwnr67hn61dzx2kwbbaw";
     })
+    # Fix build with gcc 8
+    (fetchpatch {
+      url = "https://salsa.debian.org/games-team/dolphin-emu/raw/9b7b4aeac1b60dcf28bdcafbed6bc498b2aeb0ad/debian/patches/03_gcc8.patch";
+      name = "03_gcc8.patch";
+      sha256 = "0nmlvy4pfwqpfyf1ynk2mcag097prjgjf9wzp0ngyzsshad1l15n";
```

Comment:
mismatches for me 
```
  wanted: sha256:0nmlvy4pfwqpfyf1ynk2mcag097prjgjf9wzp0ngyzsshad1l15n
  got:    sha256:1da95gb8c95kd5cjhdvg19cv2z863lj3va5gx3bqc7g8r36glqxr
```

Reply:
Best guess: a copy and paste gone wrong. Thanks @worldofpeace!